cypress/integration/configuration/shippingMethods/editShippingMethod.js

```diff
@@ -19,7 +19,7 @@ import {
 } from "../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Edit shipping method", () => {
+  describe("As a user I should be able to update and delete shipping method", () => {
```

Comment:
It seems that all found issues are similar in nature 
![image](https://user-images.githubusercontent.com/97674352/153212833-c623a1a0-4104-40e2-990e-db06be934813.png)